src/WebJobs.Script/Binding/ApiHubScriptBindingProvider.cs

```diff
@@ -71,12 +71,8 @@ public override bool TryResolveAssembly(string assemblyName, out Assembly assemb
         {
             assembly = null;
 
-            if (string.Compare(assemblyName, "Microsoft.Azure.ApiHub.Sdk", StringComparison.OrdinalIgnoreCase) == 0)
-            {
-                assembly = typeof(MetadataInfo).Assembly;
-            }
-
-            return assembly != null;
+            return Utility.TryMatchAssembly(assemblyName, typeof(MetadataInfo), out assembly) ||
+                   Utility.TryMatchAssembly(assemblyName, typeof(ApiHubFileAttribute), out assembly);
```

Comment:
For all of the ScriptBindingProviders, we now explicitly also resolve the home extension assembly